server/pulp/server/async/tasks.py

```diff
@@ -344,12 +348,15 @@ def apply_async(self, *args, **kwargs):
         :param tags:        A list of tags (strings) to place onto the task, used for searching for
                             tasks by tag
         :type  tags:        list
+        :param group_id:    The id that identifies which group of tasks a task belongs to
+        :type group_id:     basestring
```

Comment:
Consider making this a uuid instead of a basestring. If it's just a basestring we'll have to write a generator, when it would be better just use UUIDs since they are great and we don't have to maintain them. It would be good to add a doc about usage here so that callers know this needs to be a UUID.